components/webui/about_ui.cc

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#include "brave/components/webui/about_ui.h"
+#include <string_view>
+
+#include "base/containers/contains.h"
+#include "base/containers/fixed_flat_set.h"
+#include "base/strings/string_split.h"
+#include "base/strings/string_util.h"
+#include "third_party/re2/src/re2/re2.h"
+
+namespace brave {
+
+constexpr char kChromeHeader[] = "Chrome URLs";
+constexpr char kBraveHeader[] = "Brave URLs";
+constexpr char kChromePagesHeader[] = "List of Chrome URLs";
+constexpr char kBravePagesHeader[] = "List of Brave URLs";
+constexpr char kChromeInternalPagesHeader[] =
+    "List of chrome://internals pages";
+constexpr char kBraveInternalPagesHeader[] = "List of brave://internals pages";
+constexpr char kChromeURLList[] = ">chrome://";
+constexpr char kBraveURLList[] = ">brave://";
+
+std::string ReplaceAboutUIChromeURLs(std::string chrome_urls) {
+  // Replace Chrome -> Brave.
+  constexpr std::string_view chrome_header = kChromeHeader;
+  constexpr std::string_view brave_header = kBraveHeader;
+  constexpr std::string_view chrome_pages_header = kChromePagesHeader;
+  constexpr std::string_view brave_pages_header = kBravePagesHeader;
+  constexpr std::string_view chrome_internal_pages_header =
+      kChromeInternalPagesHeader;
+  constexpr std::string_view brave_internal_pages_header =
+      kBraveInternalPagesHeader;
+  constexpr std::string_view chrome_url_list = kChromeURLList;
+  constexpr std::string_view brave_url_list = kBraveURLList;
```

Comment:
I did it how it's done in every other file I came across so that's why I moved it.

Fixed in https://github.com/brave/brave-core/pull/20929/commits/28844bde3166f309f9a3e7c9b72c59646c65d4fa